packages/cohort/src/replicator/core.rs

```diff
@@ -0,0 +1,67 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use talos_certifier::model::DecisionMessage;
```

Comment:
@gk-kindred we might need to reduce copies of classes which represent the same things like decision message, candidate message. 

We already have two copies - one inside certifier, the other is inside agent. Now [the third is coming in](https://github.com/kindredgroup/talos/pull/35/files#diff-b80d6ec230a253a3fe5b04ec3ec5229d9e207c94004e59768d13767c34c10bdeR45). Given that cohort is already including agent as dependency (which comes with its data model) I am wondering does it make sense to additionally include certifier data model dependency?

Reply:
Yeah this was temporary as I needed `statemap` field which `agent` and `certifier` version of candidate message didn't have yet. Was bit unsure if I would break anything so was holding off till rest of the functionality was added in